core/common/src/test/java/alluxio/util/io/PathUtilsTest.java

```diff
@@ -507,4 +507,31 @@ public void normalizePath() throws Exception {
     assertEquals("/foo/bar//", PathUtils.normalizePath("/foo/bar//", "/"));
     assertEquals("/foo/bar%", PathUtils.normalizePath("/foo/bar", "%"));
   }
+
+  /**
+   * Tests the {@link PathUtils#getPossibleMountPoints(String)} method to
+   * throw an exception in case the path is invalid.
+   */
+  @Test
+  public void getPossibleMountPointsException() throws InvalidPathException {
+    mException.expect(InvalidPathException.class);
+    PathUtils.getPossibleMountPoints("");
+  }
+
+  /**
+   * Tests the {@link PathUtils#getPossibleMountPoints(String)} method.
+   */
+  @Test
+  public void getPossibleMountPointsNoException() throws InvalidPathException {
```

Comment:
what about adding some tests like /a/, /a/b/ and / etc

Reply:
Added